projects/go-lib/src/lib/components/go-accordion/go-accordion.component.ts

```diff
@@ -57,8 +57,12 @@ export class GoAccordionComponent implements OnInit, AfterContentInit {
     panel.expanded = false;
   }
   
-  setActiveTheme() {
-    this.activeTheme = 'go-accordion--theme-' + this.theme;
-  }
+  accordionClasses() {
+    const theme = 'go-accordion--theme-' + this.theme;
 
+    return {
+      'go-accordion--theme-light': true,
```

Comment:
We should maybe just apply the class to the html itself. Since we don't do dark themes anyways